src/components/ArticleContainer.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState, useContext } from "react";
+import { Route, Routes, Link } from "react-router-dom";
```

Comment:
As previous comment states, don't just paste imports in you're not using. Bring them in as you need them. In some cases unused imports flag as warnings, and if you're on strict mode at build time those warnings are errors and will fail the build